torch/csrc/inductor/aoti_torch/c/shim.h

```diff
@@ -141,6 +141,17 @@ AOTI_TORCH_EXPORT AOTI_TORCH_NOINLINE AOTITorchError aoti_torch_empty_strided(
     int32_t device_type,
     int32_t device_index);
 
+AOTI_TORCH_EXPORT AOTI_TORCH_NOINLINE AOTITorchError aoti_torch_from_blob(
```

Comment:
nit - ```aoti_torch``` seems to be too general? What about ```aoti_tensor_from_blob```?

Reply:
`aoti_torch_` is the uniform prefix for all the functions provided by `shim.h`. I can make it `aoti_torch_tensor_from_blob`.